prow/genjobs/cmd/genjobs/main.go

```diff
@@ -152,6 +153,7 @@ func (o *options) parseOpts() {
 	flag.BoolVar(&o.Resolve, "resolve", false, "Resolve and expand values for presets in generated job(s).")
 	flag.BoolVar(&o.SSHClone, "ssh-clone", false, "Enable a clone of the git repository over ssh.")
 	flag.BoolVar(&o.OverrideSelector, "override-selector", false, "The existing node selector will be overridden rather than added to.")
+	flag.BoolVar(&o.Internal, "internal", false, "Generate Prow jobs for the internal repos.")
```

Comment:
I am not sure what `internal` means. Maybe this would make sense as something like "support gerrit reporting" (bool) or "report label" (string)?

Reply:
You are right, `internal` is a pretty confusing word. I have created https://github.com/istio/test-infra/pull/2839 to change to `support-gerrit-reporting` as you suggested.